crates/github-actions-expressions/src/lib.rs

```diff
@@ -674,6 +674,346 @@ impl From<bool> for Expr<'_> {
     }
 }
 
+/// The result of evaluating a GitHub Actions expression.
+///
+/// This type represents the possible values that can result from evaluating
+/// constant-reducible GitHub Actions expressions.
+#[derive(Debug, Clone, PartialEq)]
+pub enum EvaluationResult {
```

Comment:
Nitpick: Can we rename this to `Evaluation`? I think that's sufficiently clear :slightly_smiling_face:

Reply:
Fixed in https://github.com/zizmorcore/zizmor/pull/1088/commits/179c19bfa5a9d451c44a895da2469c9c5616f503.